telemetry/telemetrybuffer.go

```diff
@@ -179,10 +202,18 @@ func (tb *TelemetryBuffer) PushData(ctx context.Context) {
 			push(report)
 			tb.mutex.Unlock()
 		case <-tb.cancel:
-			log.Logf("[Telemetry] server cancel event")
+			if tb.logger != nil {
+				tb.logger.Info("[Telemetry] server cancel event")
+			} else {
+				log.Logf("[Telemetry] server cancel event")
+			}
 			return
 		case <-ctx.Done():
-			log.Logf("[Telemetry] received context done event")
+			if tb.logger != nil {
+				tb.logger.Info("[Telemetry] received context done event")
```

Comment:
Any reason why we are adding `[Telemetry]` prefix to this info log ? Let's keep it consistent with other info logs for telemetry ? I assume component should tell us now that this is a telemetry log ?